### ISSUE TITLE
can't evaluate field TrimSpace in type interface {}

### ISSUE THREAD

**Issue by ikegentz** [chaoming/hugo-saasify-theme#2]
When first firing up the demo site, you'll get this error. Here's a full trace:

```
Error: error building site: render: failed to render pages: render of "home" failed: "<my-project>/themes/hugo-saasify-theme/layouts/index.html:2:3": execute of template failed: template: index.html:2:3: executing "main" at <.Content>: error calling Content: "<my-project>/content/_index.md:44:6": failed to render shortcode "features-section": "<my-project>/content/_index.md:1:1": failed to render shortcode "feature": failed to process shortcode: "<my-project>/themes/hugo-saasify-theme/layouts/shortcodes/feature.html:39:42": execute of template failed: template: shortcodes/feature.html:39:42: executing "shortcodes/feature.html" at <strings>: can't evaluate field TrimSpace in type interface {}
```

Haven't dug into this yet but for the moment but you can get the site to load by replacing ` . | default "" | strings.TrimSpace ` with ` . | default "" `

Thanks for making such an awesome theme, loving it so far!

**Comment by ikegentz** [comment]
This can be replicated simply by creating a new empty project, and copying the example site. Upon running with `npm start` this error will appear and the site will fail to render

**Comment by chaoming** [comment]
The issue is fixed in commit 60c5c47750cab52a51db734b5660f7a797d53998
